cogito/core/app.py

```diff
@@ -193,8 +195,8 @@ async def setup(self, app: FastAPI):
     def run(self):
         uvicorn.run(
             self.app,
-            host=self.config.cogito.server.fastapi.host,
-            port=self.config.cogito.server.fastapi.port,
+            host=self.config.cogito.get_fastapi_host,
```

Comment:
Shouldn’t it be used this way?

```suggestion
            host=self.config.cogito.get_fastapi_host(),
```

Reply:
Every get function is a property. In the beginning, I was trying to use the same name of the property and make the real property private to the class, but this was an error because it didn't pass to a YAML file then. So, quickly, I changed the name of every property with the prefix get_ and made the real properties public again. 

I know that the name of that property can make anyone think it’s a method. If it's confusing, we can change everything to a method instead.